extensions/spark/src/main/scala/ai/djl/spark/SparkModel.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.spark
+
+import ai.djl.inference.Predictor
+import ai.djl.modality.Classifications
+import ai.djl.repository.zoo.{Criteria, ZooModel}
+import ai.djl.training.util.ProgressBar
+import ai.djl.translate.Translator
+import org.apache.spark.sql.Row
+
+import java.io.Serializable
+
+/**
+ * `SparkModel` is the implementation of [[ai.djl.Model]] for Spark support.
```

Comment:
does scala doc support markdown?

Reply:
Yes. This is the scala format of scaladoc